glslc/test/option_dash_x.py

```diff
@@ -57,6 +60,17 @@ class TestDashXHlslOnGlslShader(expect.ErrorMessageSubstr):
 
     shader = FileShader(GLSL_VERTEX_SHADER, '.vert')
     glslc_args = ['-x', 'hlsl', '-c', shader]
+    # This is not a terribly good error message since HLSL doesn't support
```

Comment:
:-) 
I actually filed https://github.com/KhronosGroup/glslang/issues/834 about the quality of the error message.  If that gets "fixed" then I'll have to react again.

Thanks for the LGTM.